dataservices/client/config.go

```diff
@@ -0,0 +1,45 @@
+package client
+
+/* CHECKLIST
+ * [ ] Uses interfaces as appropriate
+ * [ ] Private package variables use underscore prefix
+ * [ ] All parameters validated
+ * [ ] All errors handled
+ * [ ] Reviewed for concurrency safety
+ * [ ] Code complete
+ * [ ] Full test coverage
+ */
+
+import (
+	"net/url"
+
+	"github.com/tidepool-org/platform/app"
+)
+
+type Config struct {
+	Address        string `json:"address"`
+	RequestTimeout int    `json:"requestTimeout"`
+}
+
+func (c *Config) Validate() error {
+	if c.Address == "" {
+		return app.Error("client", "address is missing")
```

Comment:
do these give enough info so we know which client? .... I guess they do in the stack trace don't they